src/libraries/Microsoft.Bcl.Memory/ref/Microsoft.Bcl.Memory.cs

```diff
@@ -1,34 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-namespace System.Buffers.Text
-{
-    public static class Base64Url
```

Comment:
@ViktorHofer, I'm confused by this change. Where does the netstandard2.0 ref for this come from now that this is deleted?

Reply:
That's the idea afaik yeah. From the linked issue:
> Since .NET 6, out-of-band reference assemblies like Microsoft.Extensions.* don't ship to customers anymore as they aren't part of the generated packages. Their sole purpose today is infrastructure only needs (ApiCompat and GenFacades).

/cc @ericstj

Reply:
I'm out on leave but just saw this message. Yes, @akoeplinger is correct. Reference assemblies that don't ship inside a shared framework don't reach the customer anymore. This started between .NET 5 and .NET 6.

Example:
https://nuget.info/packages/System.Data.Odbc/5.0.0 (has a "ref" folder)
https://nuget.info/packages/System.Data.Odbc/8.0.0 (doesn't have a "ref" folder)

Therefore we don't need to even let Roslyn generate them.

For libraries that ship inside any of our available shared frameworks (Microsoft.NETCore.App.Ref, Microsoft.AspNetCore.App.Ref and Microsoft.WindowsDesktop.App.Ref) we need reference assemblies. The idea here is that we let the compiler automatically generate them from the implementation assembly instead of us hand authoring them via dedicated ref projects (https://github.com/dotnet/runtime/issues/58163).